recipes/libmatio/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.5.6" %}
+package:
+  name: libmatio
+  version: {{ version }}
+
+source:
+  fn: matio-{{ version }}.tar.gz
+  url: https://github.com/tbeu/matio/releases/download/v{{ version }}/matio-{{ version }}.tar.gz
+  md5: bb53fd06f7ebb74589e9dd684b1b66a6
+
+build:
+  number: 0
+  skip: true     # [win]
+
+requirements:
+  build:
+    - autoconf
+    - automake
+    - libtool
+    - gcc
+    - hdf5
+    - zlib
+
+  run:
+    - hdf5
+    - zlib
+
+test:
+  commands:
+    - test -e $PREFIX/include/matio.h  # [unix]
```

Comment:
Are there any libraries that matio is generating? Could we please test to see if they are present, as well?